raddb/mods-available/ldap

```diff
@@ -394,190 +506,262 @@ ldap {
 		#  controls.
 		#
-		#  These encode the NAS-IP-Address/NAS-IPv6-Address,
-		#  User-Name, Acct-Session-ID, Acct-Multi-Session-ID
-		#  as session tracking controls, in applicable LDAP operations.
-		#  Default 'no'.
+		#  These encode the `NAS-IP-Address`, `NAS-IPv6-Address`, `User-Name`, `Acct-Session-Id`, 
+		#  `Acct-Multi-Session-Id` as session tracking controls, in applicable LDAP operations.
+		#
+		#  Default `no`
 		#
 #		session_tracking = yes
 
 		#
-		#  SASL Security Properties (see SASL_SECPROPS in ldap.conf man page).
-		#  Note - uncomment when using GSS-API sasl mechanism along with TLS
-		#  encryption against Active-Directory LDAP servers (this disables
-		#  sealing and signing at the GSS level as required by AD).
+		#  sasl_secprops:: SASL Security Properties (see SASL_SECPROPS in ldap.conf man page).
+		#
+		#  NOTE: uncomment when using GSS-API sasl mechanism along with TLS encryption against
+		#  Active-Directory LDAP servers (this disables sealing and signing at the GSS level as
+		#  required by AD).
 		#
 #		sasl_secprops = 'noanonymous,noplain,maxssf=0'
 
-		#  Seconds to wait for LDAP query to finish. default: 20
+		#
+		#  res_timeout:: Seconds to wait for LDAP query to finish.
+		#
+		#  Default: `20`
+		#
 		res_timeout = 10
 
-		#  Seconds LDAP server has to process the query (server-side
-		#  time limit). default: 20
 		#
-		#  LDAP_OPT_TIMELIMIT is set to this value.
+		#  srv_timelimit:: Seconds LDAP server has to process the query (server-side
+		#  time limit).
+		#
+		#  Default: `20`
+		#
+		#  `LDAP_OPT_TIMELIMIT` is set to this value.
+		#
 		srv_timelimit = 3
 
-		#  LDAP_OPT_X_KEEPALIVE_IDLE
+		#
+		#  idle:: Set the number of seconds a connection needs to remain idle
+		#  before  TCP starts sending keepalive probes.
+		#
+		#  `LDAP_OPT_X_KEEPALIVE_IDLE` is set to this value.
+		#
 		idle = 60
 
-		#  LDAP_OPT_X_KEEPALIVE_PROBES
+		#
+		#  probles:: Set the maximum number of keepalive probes TCP should send
+		#  before dropping  the  connection.
+		#
+        #  `LDAP_OPT_X_KEEPALIVE_PROBES` is set to this value.
+        #
 		probes = 3
 
-		#  LDAP_OPT_X_KEEPALIVE_INTERVAL
+		#
+		#  interval:: Set the interval in seconds between individual keepalive probes.
+		#
+		#  `LDAP_OPT_X_KEEPALIVE_INTERVAL` is set to this value.
+		#
 		interval = 3
 
-		#  ldap_debug: debug flag for LDAP SDK
-		#  (see OpenLDAP documentation).  Set this to enable
-		#  huge amounts of LDAP debugging on the screen.
-		#  You should only use this if you are an LDAP expert.
 		#
-		#	default: 0x0000 (no debugging messages)
-		#	Example:(LDAP_DEBUG_FILTER+LDAP_DEBUG_CONNS)
+		#  ldap_debug:: Debug flag for LDAP SDK (see OpenLDAP documentation).
+		#  Set this to enable huge amounts of LDAP debugging on the screen.
+		#
+		#  TIP: You should only use this if you are an LDAP expert.
+		#
+		#  Default: 0x0000 (no debugging messages)
+		#
+		#  Example: (`LDAP_DEBUG_FILTER` + `LDAP_DEBUG_CONNS`)
```

Comment:
It would be kind of useful to include a table here for the different levels if it's not too big.